zkevm-circuits/src/evm_circuit/witness.rs

```diff
@@ -63,7 +63,10 @@ impl<F: FieldExt> BlockContext<F> {
                 [
                     F::from(BlockContextFieldTag::Time as u64),
                     F::zero(),
-                    F::from(self.time),
+                    RandomLinearCombination::random_linear_combine(
+                        self.time.to_le_bytes(),
+                        randomness,
+                    ),
```

Comment:
why this change? the field is big enough to contain u64?

Reply:
thanks for pointing this out. I updated the gadget so that this change isn't needed anymore.